README.md

```diff
@@ -1,130 +1,51 @@
-* **Sidebar buttons:** Stay up-to-date with how many reviews, assignments, and open pull requests you have with buttons in the Mattermost sidebar.
-* **Slash commands:** Interact with the Bitbucket plugin using the `/bitbucket` slash command.
-
-![Bitbucket plugin screenshot](https://user-images.githubusercontent.com/45372453/97643091-114a1500-1a47-11eb-9863-2e0e308706ea.png)
-
-## Admin guide
-
-This guide is intended for Mattermost System Admins setting up the Bitbucket plugin and Mattermost users who want information about the plugin functionality. For more information about contributing to this plugin, visit the Development section. The Mattermost Bitbucket plugin uses a webhook to connect your Bitbucket account to Mattermost to listen for incoming Bitbucket events. Events notifications are via Direct Message in Mattermost. The Events don’t need separate configuration.
-
-### Prerequisites
-
-This guide assumes:
-
-* You have a Bitbucket account.
-* You're a Mattermost System Admin.
-* You're running Mattermost Server v5.25 or higher.
-
-### Installation
-
-#### Marketplace installation
-
-1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
-2. Search for "Bitbucket" or find the plugin from the list.
-3. Select **Install**.
-4. When the plugin has downloaded and been installed, select **Configure**.
-
-#### Manual installation
-
-If your server doesn't have access to the internet, you can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-bitbucket/releases) and upload it to your server via **System Console > Plugin Management**. The releases on this page are the same used by the Marketplace. To learn more about how to upload a plugin, see [the documentation](https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/).
+[![Build Status](https://github.com/mattermost/mattermost-plugin-bitbucket/actions/workflows/ci.yml/badge.svg)](https://github.com/mattermost/mattermost-plugin-bitbucket/actions/workflows/ci.yml)
+[![Latest Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-bitbucket?style=flat-square)](https://github.com/mattermost/mattermost-plugin-bitbucket/releases)
+[![Help Wanted](https://img.shields.io/github/issues/mattermost/mattermost-plugin-bitbucket?style=flat-square&color=brightgreen&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-bitbucket/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22)
 
-### Configuration
+## Help wanted tickets [[here]](https://github.com/mattermost/mattermost-plugin-bitbucket/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22)
 
-Configuration is started in Bitbucket and completed in Mattermost.
+## Contents
 
-#### Step 1: Register an OAuth application in Bitbucket
+- [Overview](#overview)
+- [Features](#features)
+- [Admin Guide\*](docs\admin-guide.md)
```

Comment:
At least on GitHub, the link is broken.
![Screenshot from 2023-11-01 12-59-58](https://github.com/mattermost/mattermost-plugin-bitbucket/assets/16541325/cdd93b2e-342a-47a4-8376-fc6eb530d8f5)

Reply:
Worked on local

Reply:
Do you use Windows locally by any chance?

Reply:
Yes I do